KroneMultimedia/SharedFastfile.rb

```diff
@@ -0,0 +1,289 @@
+end
+
+desc "Run Unit/UI Tests"
+lane :tests do
+  certificates
+
+  kill_simulator
+  snapshot( {
+      workspace: $self_workspace,
+      scheme: $self_scheme,
+      devices: [ "iPhone 6" ],
+      languages: [ "en-US" ],
+      clear_previous_screenshots: true,
+      reinstall_app: true,
+      clean: true,
+      buildlog_path: "fastlane/logs",
+      skip_open_summary: true
+  })
+  sh "cat logs/*|xcpretty --report html --output logs/test_report.html  -t 2>&1 >> /dev/null"
+  kill_simulator
+end
+
+lane :certificates do
+
+  $self_idfs.each do |idf|
+     match(git_url: "ssh://git@certs/mobile/IOS-CERTIFICATES.git", 
+        type: "development", 
+        app_identifier: idf)
+
+     match(git_url: "ssh://git@certs/mobile/IOS-CERTIFICATES.git", 
+        type: "appstore", 
+        app_identifier: idf)
+
+ end
+
+end
+
+lane :build_ipa do
+   before_build
+   certificates
+
+  gym({
+    scheme: $self_scheme,
+    clean: true,
+    workspace: $self_workspace,
+    silent: true,
+    xcargs: "GCC_PREPROCESSOR_DEFINITIONS='CMD_LINE_BUILD'",
+    buildlog_path: "fastlane/logs"
+
+  })
+
+end
+
+lane :upload_testflight do
+  pilot( {
+    skip_submission: true
+  })
+end
+
+
+lane :testflight do
+    build_ipa
+    upload_testflight
+end
+
+
+
+
+lane :bump_version do
+    ensure_git_status_clean
+    version = prompt(text: "New Version number?")
+    increment_version_number(version_number: version)
+    commit_version_bump(message: "Version Bump!!")
+end
+
+
+lane :submit_build do |options| 
+  require 'spaceship'
+  Spaceship::Tunes.login
+  app = Spaceship::Tunes::Application.find($self_idfs[0])
+  
+
+  version = app.edit_version
+
+  
+    #get latest build
+    use_build = version.candidate_builds.first
+
+
+    # OR With this one set a specific
+    #get specific build
+
+    if options[:build_number]
+      puts "Using fixed nr: #{options[:build_number]}"
+      use_build = version.candidate_builds.find({ build_version: options[:build_number] }).first
+    end
+
+    puts "Using #{use_build}"
+
+
+    version.select_build(use_build)
+
+    release_notes = prompt(text: "Release Notes (end with END):", multi_line_end_keyword: "END")
+
+    version.release_on_approval = true
+    version.can_beta_test = true
+    version.release_notes["de-DE"] = release_notes
+    version.save!
+
+    submission = app.create_submission
+    # Set app submission information
+    submission.content_rights_contains_third_party_content = true
+    submission.content_rights_has_rights = true
+    
+
+   submission.export_compliance_available_on_french_store = false
+   submission.export_compliance_contains_proprietary_cryptography = false
+   submission.export_compliance_contains_third_party_cryptography = false
+   submission.export_compliance_is_exempt = false
+   submission.export_compliance_uses_encryption = false
+   submission.export_compliance_app_type = nil
+   submission.export_compliance_encryption_updated = false
```

Comment:
This should actually be part of `deliver`, but I guess you had troubles using `deliver` to submit your app for Review? Is this lane just to submit an existing build for Review, right? 
What do you think about adding this functionality to `deliver`?